osgp/platform/osgp-adapter-ws-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/ws/da/application/mapping/DistributionAutomationMapper.java

```diff
@@ -35,58 +36,77 @@
     public void configure(final MapperFactory mapperFactory) {
         mapperFactory.getConverterFactory().registerConverter(new PassThroughConverter(DateTime.class));
         mapperFactory.getConverterFactory().registerConverter(new XMLGregorianCalendarToDateTimeConverter());
+        mapperFactory.getConverterFactory().registerConverter(new XMLGregorianCalendarToInstantConverter());
```

Comment:
It would almost have been nice had you placed a comment with this line like "Look here for the actual change". ;)
If the rest besides this (and the import that belongs with it) contains changes I should have a look at, please let me know.

Reply:
Indeed, registration of the new converter is the only change in this file...